codice/ddf
# DDF-1957: Create backend for Log Service

#### What does this PR do?
Provides a backend logging Jolokia endpoint
#### Who is reviewing it?
@garrettfreibott @lessarderic @Lambeaux @bantillo @oconnormi 
#### How should this be tested?
 1) Start DDF
 2) Run through the installer
 3) Verify logs show up when hitting endpoint: https://localhost:8993/jolokia/exec/org.codice.ddf.logging.platform.LoggingServiceBean:service=logging-service/retrieveLogEvents
5) Verify "number of logs to store" can be changed via the Admin Console: DDF Platform -> Configuration -> Logging Service
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-1957
#### Screenshots (if appropriate)
#### Checklist:
